plugwise/smile.py

```diff
@@ -7,7 +7,7 @@
 import aiohttp
 import async_timeout
 from dateutil.parser import parse
-from lxml import etree
+from defusedxml import ElementTree as etree
```

Comment:
Experimental - though it seems to work fine (@CoMPaTech @bouwew)

Reply:
I can confirm.